ckanext/canada/validators.py

```diff
@@ -110,6 +110,17 @@ def canada_validate_generate_uuid(value):
         raise Invalid(_("Badly formed hexadecimal UUID string"))
 
 
+email_pattern = re.compile(r"^[A-Za-z0-9\.\+_-]+@[A-Za-z0-9\._-]+\.[a-zA-Z]*$")
```

Comment:
I suggest https://html.spec.whatwg.org/#e-mail-state-(type=email) (the regexp is below) and add a comment that links to this doc to say where we got it from.